pkg/apiserver/profiling/model.go

```diff
@@ -84,11 +100,15 @@ func NewTask(ctx context.Context, taskGroup *TaskGroup, target model.RequestTarg
 }
 
 func (t *Task) run() {
-	fileNameWithoutExt := fmt.Sprintf("profiling_%d_%d_%s", t.TaskGroupID, t.ID, t.Target.FileName())
-	protoFilePath, rawDataType, err := profileAndWritePprof(t.ctx, t.fetchers, &t.Target, fileNameWithoutExt, t.taskGroup.ProfileDurationSecs)
+	fileNameWithoutExt := fmt.Sprintf("profiling_%d_%d_%s_%s", t.TaskGroupID, t.ID, t.ProfilingType, t.Target.FileName())
+	protoFilePath, rawDataType, err := profileAndWritePprof(t.ctx, t.fetchers, &t.Target, fileNameWithoutExt, t.taskGroup.ProfileDurationSecs, t.ProfilingType)
 	if err != nil {
 		t.Error = err.Error()
```

Comment:
When task is skipped, maybe there is no need to set `t.Error`, since it is a normal and expected state, instead of a actual error state.